microsoft/testsuites/kdump/kdumpcrash.py

```diff
@@ -203,10 +206,31 @@ def kdumpcrash_validate_large_memory_auto_size(
         self.crash_kernel = "auto"
         self._kdump_test(node, log_path, log)
 
+    def _get_vmbus_version(self, node: Node) -> VersionInfo:
+        vmbus_version_pattern = re.compile(
```

Comment:
Check with @LiliDeng what's the best way to get vmbus version with existing tools.

Reply:
Baihua is trying to fix when dmesg is full and flushed by many noisy message, at that time, it can't get vmbus version from dmesg.

Reply:
Sure, have done.